### DISCUSSION TITLE
socks5 outbound在任何情况下都会发送域名吗?

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [apernet/hysteria#933] [Q&A]
socks5 outbound在任何情况下都会发送域名吗?

**Comment by haruue** [comment]
仅在 hysteria 客户端通过 socks5/http 收到带域名的请求的情况下， hysteria 服务端才会向 socks5 outbound 发送域名。

请注意 hysteria 并不支持 sniffer 功能， 也就是说， 如果你在客户端侧使用 tproxy/redirect/tun2socks 配置成透明代理， 并且没有加入任何支持 sniffer 的工具， 那么服务端会向 socks5 outbound 发送 IP 地址而不是域名请求。

**Reply by CCCAUCHY** [reply]
是否可以将匹配ACL IP规则时解析出的结果用于socks5 outbound?
